migrations/20190325115901_create-notifications-config.js

```diff
@@ -0,0 +1,13 @@
+exports.up = knex =>
+  Promise.all([
+    knex.schema.createTable('notifications_config', table => {
+      table.increments('id').primary('pk_notifications')
+      table.string('email').notNullable()
+      table.string('establishment').notNullable()
+      table.string('role').notNullable()
+      table.string('name').nullable()
+      table.unique(['email', 'establishment', 'role'])
+    }),
+  ])
```

Comment:
How is this applied to the database? And if you needed to change the structure of a table, how might you do this without needing to recreate the table?

Reply:
@timharrison-moj this is a migration file.. and if you needed to change a the table structure you'd have to create a new migration file with an alter command... something like `knex.schema.alterTable('xyz')`
The way migrations work is that it creates a migrations table which stores the last migration that was run.
If it's been run already run it won't run it again... so you don't actually have to recreate the table..

Reply:
We use knex.js for migrations. Future migrations would be non-destructive.
Small data set now but added index for future